webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -116,7 +117,7 @@ const NavigationSideBar: React.FC = () => {
   const nodeRef = useRef(null);
 
   const [showSettings, setShowSettings] = useState<boolean>(false);
-
+  const GASLinkOn = useFeature().isOn('GAS');
```

Comment:
nit: we usually reserve PascalCase for types and classes, could you change this?
```suggestion
  const gasLinkOn = useFeature().isOn('GAS');
```